### PR TITLE
Decrease linter errors, keep commented tests warnings

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-runtime/src/services/uppyService.ts [hunk 1]
```diff
@@ -35,16 +35,13 @@ export class UppyService {
...
-    tusExtension,
     onBeforeRequest
   }: {
     tusMaxChunkSize: number
     tusHttpMethodOverride: boolean
-    tusExtension: string
     onBeforeRequest: () => void
   }) {
     const chunkSize = tusMaxChunkSize || Infinity
-    const uploadDataDuringCreation = tusExtension.includes('creation-with-upload')
```
Comment (JammingBen) [inline packages/web-runtime/src/services/uppyService.ts:47]: What happened here?

Comment (JanAckermann) [reply packages/web-runtime/src/services/uppyService.ts:47]: nothing was done with uploadDataDuringCreation var

Comment (JammingBen) [reply packages/web-runtime/src/services/uppyService.ts:47]: W00t, yes. See line 54 (line 51 now).

Comment (kulmann) [reply packages/web-runtime/src/services/uppyService.ts:47]: ... and that's why the build is failing, see https://drone.owncloud.com/owncloud/web/28037/8/6

Comment (JanAckermann) [reply packages/web-runtime/src/services/uppyService.ts:47]: Oh yes, had old master, resetting

**Review by kulmann — APPROVED** [review]

**Pull request merged**